ccmbioinfo/lfs
# CARDS-1932: Update PREMs email templates

This includes [CARDS-1984](https://phenotips.atlassian.net/browse/CARDS-1984), [CARDS-1932](https://phenotips.atlassian.net/browse/CARDS-1932), and [CARDS-1967](https://phenotips.atlassian.net/browse/CARDS-1967).

To test:

- build with `-Pdocker`
- in `compose-cluster`, run `python3 generate_compose_yaml.py --dev_docker_image --oak_filesystem --cards_project cards4prems --smtps --smtps_test_container --smtps_test_mail_path ~/cardsmail --composum`
- edit the generated `docker-compose.yml` file, and at the end of the environment section of the cardsinitial service, add:
```
    - NIGHTLY_NOTIFICATIONS_SCHEDULE=0 * * * * ? *
    - TZ=America/Toronto
```
- start it with `docker-compose up`
- create a new patient information, and visit informations 7, 9, 11 days ago.
- wait one minute, then check the contents of the `~/cardsmail` folder


[CARDS-1984]: https://phenotips.atlassian.net/browse/CARDS-1984?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[CARDS-1932]: https://phenotips.atlassian.net/browse/CARDS-1932?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[CARDS-1967]: https://phenotips.atlassian.net/browse/CARDS-1967?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
